modules/sc-mesh-secure-deployment/src/nats/scripts/config.py

```diff
@@ -8,12 +8,12 @@
     print("No identity found!!!!!!!!!!!!!!!!!!!!!!!!")
     print("Please run _cli_command_get_identity.py to get the identity (creates identity.py)")
 
-MODULE_IP = "10.10.10.2"  # or 192.168.1.x - brlan ip address
+MODULE_IP = "10.10.20.2"  # or 192.168.1.x - brlan ip address
```

Comment:
Please remove all IPv4

Reply:
config.py is the configuration file for testing scripts.  As we have IPv4 currently in multiradio --> it is needed.   Cleanup will be done when IPv6 will be introduced.

Reply:
Sure, this is an issue but as with a lot of stuff at the moment, I think this is a "known issue" that needs to be logged as a separate ticket in the MS Dev Jira and picked up as a separate PR.

Reply:
That connection is for identity request.  You are just connecting to device which is connected with Ethernet cable.  Later, if you need to send messages to other device there should be used bridge IP and 10.10.20.2 as a gateway.